client/templates/profilelink/profilelink.js

```diff
@@ -0,0 +1,18 @@
+if (Meteor.isClient) {
+    Template.profilelink.helpers({
+        signedInAs: function() {
+            var user = Meteor.user();
+            if (user) {
```

Comment:
I think we want to be able to access this in multiple places, can we abstract this logic somewhere?